packages/components/package.json

```diff
@@ -69,6 +69,7 @@
     "react-helsinki-headless-cms": "1.0.0-alpha83",
     "react-i18next": "11.18.6",
     "react-leaflet": "4.2.0",
+    "react-toastify": "^9.0.3",
```

Comment:
Where is the toastify needed?

Reply:
`yarn check:install` complained about it for some reason, Something was wrong in the main branch?